tasks.php

```diff
@@ -887,6 +891,24 @@ function handle_action_on_a_specified_task()
         // Redirect back to show task page to clear POST data
         metarefresh(0, "$tasks_url?action=show&task_id=$task_id");
     }
+    else if ($action == 'save_edit_comment') {
+      $comment_id = array_get($_POST, 'comment_id', "");
+      $comment = trim(array_get($_POST, 'task_comment', ''));
+      [$u_id, $comment_date] = explode('_', $comment_id, 2);
+      if ($u_id === $requester_u_id) {
```

Comment:
We should enable SAs to edit comments as well. While SAs do this very rarely, right now we need to go into the database to do it. That would change this to be:
```
if ($u_id === $requester_u_id || user_is_a_sitemanager()) {
```